File: Data Structures & Algorithms/guess-number-higher-or-lower/submission-0.java
/** 
 * Forward declaration of guess API.
 * @param  num   your guess
 * @return 	     -1 if num is higher than the picked number
 *			      1 if num is lower than the picked number
 *               otherwise return 0
 * int guess(int num);
 */

public class Solution extends GuessGame {
    public int guessNumber(int n) {
        int start = 1;
        int m = 0;
        while(start <= n) {
            m = start + (n - start)/2;
            if(guess(m) == 1) {
                start = m + 1;
            }else if(guess(m) == -1) {
                n = m - 1;
            }else {
                return m;
            }
        }
        return m;
    }
}

File: Data Structures & Algorithms/remove-element/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} val
     * @return {number}
     */
    removeElement(nums, val) {
        let l = 0;
        for(let i = 0 ; i < nums.length; i++){
            if(nums[i] != val) {
                nums[l++] = nums[i];
            }
        }
        return l;
    }
}
